bodhi/tests/server/services/test_releases.py

```diff
@@ -229,6 +231,7 @@ def test_new_release_invalid_tags(self):
                  "candidate_tag": "epel42-candidate",
                  "override_tag": "epel42-override",
                  "csrf_token": self.get_csrf_token(),
+                 "mail_template": "fedora_errata_template"
```

Comment:
Why add ```mail_template``` to all these existing tests?

Reply:
Current logic requires ```mail_template``` while creating new release. Should we add default template ```fedora_errata_template``` when it is not provided?